src/fillTank.test.js

```diff
@@ -1,11 +1,109 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(fillTank)
+      .toBeInstanceOf(Function);
+  });
+
+  it(`should order full tank if 'amount' is not given`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+    const expectedResult = 40;
+
+    fillTank(customer, 4);
+
+    expect(customer.vehicle.fuelRemains).toBe(expectedResult);
+  });
+
+  it(`should order full tank if 'amount'
+    is greater than the tank can accommodate`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+    const expectedResult = 40;
+
+    fillTank(customer, 4, 90);
```

Comment:
The test case description suggests that the 'amount' is greater than the tank can accommodate, but the function call `fillTank(customer, 4, 90)` includes an 'amount' of 4, which is not greater than the tank capacity. Consider revising the 'amount' parameter to reflect the test case description accurately.